### PR TITLE
MCS-258 Return Object Shape and Color in Object Metadata

### PR DESCRIPTION
**Pull request by ThomasSchellenbergNextCentury**
Corresponding Unity PR https://github.com/NextCenturyCorporation/ai2thor/pull/43

### CODE REVIEW

**Comment by rartiss55** [comment]
Built the associated unity branch and ran this with the playroom...noticed the changing table shelves didn't have colors listed:
Screen Shot 2020-06-11 at 1 07 17 PM

File: scene_generator/objects.py [hunk 1]
```diff
@@ -283,8 +283,8 @@
         "z": 2
     }
 }, {
-    # Readers, please ignore the "yellow number 1" in the type: the object's chosen material will change this design.
-    "type": "block_yellow_number_1",
+    # Readers, please ignore the "blue letter c" in the type: the object's chosen material will change this design.
+    "type": "block_blue_letter_c",
```
Comment (rartiss55) [inline scene_generator/objects.py:287]: Silly question but: why did this change?

**Comment by ThomasSchellenbergNextCentury** [comment]
Re changing table shelves: Yeah, that's more complicated, but fortunately the changing table's shelves are never going to be the Target Object in a Goal, so I'm not in as much of a rush on that.

File: scene_generator/objects.py (hunk 1 above)
Comment (ThomasSchellenbergNextCentury) [reply scene_generator/objects.py:287]: An object's "shape" word is dependent on its specific type, so the type here needs to be a "number block" to match its description for the scene generator.

**Pull request merged**